client/lib/cart/store/index.js

```diff
@@ -21,14 +21,10 @@ import {
 	CART_TAX_POSTAL_CODE_SET,
 	CART_RELOAD,
 } from 'calypso/lib/cart/action-types';
-import {
-	TRANSACTION_NEW_CREDIT_CARD_DETAILS_SET,
-	TRANSACTION_PAYMENT_SET,
```

Comment:
This string appears in lib/redux-bridge. Seems related.

Reply:
Yes... I think those actions are copying the country code into the redux store for some purpose. That should also all probably be removed, but it's a bit tangled and I don't want to dive into that just yet.